### ISSUE TITLE
Local variable 'vectors' referenced before assignment

### ISSUE THREAD

**Issue by hpniemi** [f5devcentral/f5-journeys#132]
### Environment
Client: Docker Engine - Community
 Version:           20.10.17
 API version:       1.41
 Go version:        go1.17.11
 Git commit:        100c701
 Built:             Mon Jun  6 23:02:56 2022
 OS/Arch:           linux/amd64
Ubuntu 18.04.4 LTS

f5-journeys-4.1.0

##### F5OS Platform and Version

```
R4600 1.5
```
##### DESTINATION BIG-IP VERSION

```
17.1

```
##### SOURCE BIG-IP VERSION

```
16.x
```

### SUMMARY

Imported UCS, went ok, no issues. Doing offline manually, no connection to systems. Trying to generate files, says immediately:

Local variable 'vectors' referenced before assignment, try again.

##### STEPS TO REPRODUCE

```

```

###### Journeys Logs:

```
ERROR 2024-03-20 12:49:26,254 handlers Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/code/journeys/backend/views.py", line 600, in output
    output_ucs_path, output_as3_path = controller.generate(
  File "/code/journeys/controller.py", line 130, in decorator
    ret = func(*args, **kwargs)
  File "/code/journeys/controller.py", line 723, in generate
    if self.conflict_handler.get_conflicts(session=self.session):
  File "/code/journeys/modifier/conflicts/handler.py", line 56, in get_conflicts
    self.run_plugins(session=session)
  File "/code/journeys/modifier/conflicts/handler.py", line 37, in run_plugins
    self.plugins = [
  File "/code/journeys/modifier/conflicts/handler.py", line 38, in <listcomp>
    cls(
  File "/code/journeys/modifier/conflicts/plugins/Spva.py", line 102, in __init__
    "vs-dos-vectors": self._find_vs_level_dos_vectors(config),
  File "/code/journeys/modifier/conflicts/plugins/Spva.py", line 300, in _find_vs_level_dos_vectors
    for vector in vectors.fields:
UnboundLocalError: local variable 'vectors' referenced before assignment
```
###### Developer console output:

```

```
###### App Version:

4.1.0

**Comment by wojtek0806** [comment]
What is the version of the APP you are using? We also need the log file and dev console output as per template? @hpniemi

**Comment by hpniemi** [comment]
[journeys.log](https://github.com/f5devcentral/f5-journeys/files/14666991/journeys.log)

**Comment by hpniemi** [comment]
f5-journeys-4.1.0

**Comment by wojtek0806** [comment]
tracking with: JOURNEYS-638

**Comment by wojtek0806** [comment]
fixed with release 4.2.0
